Packages/macOS/CmuxTerminal/Tests/GhosttyRuntimeTestStubs/GhosttyRuntimeTestStubs.c

```
@@ -331,7 +453,19 @@ void ghostty_surface_free(void *surface) {
         cmux_test_font_callback_userdata = NULL;
     }
 }
-void ghostty_surface_free_text(void) {}
+void ghostty_surface_request_process_termination(void *surface) {
+    (void)surface;
+}
+void ghostty_surface_free_text(void *surface, ghostty_text_s *text) {
```

Cppcheck (2.21.0)

[style] The function 'ghostty_surface_free_text' is never used.

(unusedFunction)